src/main/resources/xml/schema/2020-1/nordic2020-1.sch

```diff
@@ -887,8 +887,18 @@
         <p>Rearnotes should not be used. Use endnotes instead.</p>
         <rule context="*">
             <let name="context" value="concat('(&lt;', name(), string-join(for $a in (@*) return concat(' ', $a/name(), '=&quot;', $a, '&quot;'), ''), '&gt;)')"/>
-            <report test="tokenize(@epub:type, '\s+') = ('rearnote', 'rearnotes')">[nordic290] Rearnotes are deprecated. Endnotes are required to be used instead. <value-of select="$context"/></report>
+            <assert test="tokenize(@epub:type, '\s+') = ('rearnote', 'rearnotes')">[nordic290] Rearnotes are deprecated. Endnotes are required to be used instead. <value-of select="$context"/></assert>

```

Comment:
Won't this require either `rearnote` or `rearnotes` on all elements? The context is `*`, and the test checks for the presence of one of the two values. It was `report` so that this would be forbidden on all elements.

Reply:
Correct, realized it and will change it in the next PR :)